### PR TITLE
Feature/fitness

### PR DESCRIPTION
**Pull request by mrco245**
This is the start of the fitness tracker feature. 

This was built upon the authentication feature, so review that one first - https://github.com/mrco245/SWENG861-Group3/pull/3 

This updates the styling for the signin/signup/profile pages, and updates the header. 

Still need to update to add unit tests and clean up a few items, but wanted to provide this earlier to give people time to review. 

Some screen shots:

image
image
image
image
image
image

### CODE REVIEW

File: client/src/components/Cadio.jsx
Comment (bagby007) [inline client/src/components/Cadio.jsx:1]: Rename to Cardio

File: client/src/components/Header.jsx [hunk 1]
```diff
@@ -1,24 +1,77 @@
...
+import { useSelector, useDispatch } from "react-redux";
+import { Navbar, Nav } from "react-bootstrap";
+import { useNavigate } from "react-router-dom";
+import { signOut } from "../redux/user/userSlice";
 
 export default function Header() {
+  const { currentUser } = useSelector((state) => state.user);
+  const location = useLocation();
+  const isHomePage = location.pathname === "/";
+  const isLoginPage = location.pathname === "/singin";
```
Comment (bagby007) [inline client/src/components/Header.jsx]: "signin"

**Review by bagby007 — COMMENTED** [review]
Everything looks great, I have a few comments on spelling but I plan to re-branch and style by friends page from this.

**Comment by stm5645** [comment]
I just saw this now and I really like the UI! I'll update mine similar to yours tomorrow.

**Review by bagby007 — APPROVED** [review]

**Pull request merged**